backend/src/main/java/quarano/actions/AnomaliesProperties.java

```diff
@@ -10,11 +12,19 @@
 /**
  * @author Oliver Drotbohm
  */
-@Getter
+@Getter(value = AccessLevel.PRIVATE)
 @RequiredArgsConstructor
 @ConstructorBinding
 @ConfigurationProperties(prefix = "quarano.anomalies")
 public class AnomaliesProperties {
 
-	private final BodyTemperature temperatureThreshold;
+	private final BodyTemperature temperatureThresholdIndex;
+	private final BodyTemperature temperatureThresholdContact;
+
+	public BodyTemperature getTemperatureThreshold(CaseType type) {
```

Comment:
Now that we use a proper version in the build, can we use `@since` tags for newly introduced, _non-private_ methods and classes? It's easier to reason about which version brought some code by that.

Reply:
done